tools/build/docker/s6/services.d/lanraragi/run

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/execlineb -P
-s6-envuidgid -B koyomi:koyomi
+#!/bin/sh
 cd /home/koyomi/lanraragi/
-perl /home/koyomi/lanraragi/script/launcher.pl -f /home/koyomi/lanraragi/script/lanraragi
+export HOME=/home/koyomi
+s6-setuidgid $(id -u koyomi):$(id -g koyomi) perl /home/koyomi/lanraragi/script/launcher.pl -f /home/koyomi/lanraragi/script/lanraragi
```

Comment:
According to the doc link you gave, wouldn't it be better to directly use `s6-setuidgid koyomi` instead of grabbing stuff with `id` ?

Reply:
Reading the docs I assumed it would only update the UID, not the GID, but testing it out that seems not to be the case. Updated.